upstream-community-operators/openshift-ibm-quantum-operator/openshift-ibm-quantum-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+  - name: aplha
+    currentCSV: ibm-quantum-operator.v0.1.1
```

Comment:
@husky-parul, this CSV name does not exists, please use `openshift-ibm-quantum-operator.v0.1.1`.